packages/core/upload/server/src/services/ai-metadata.ts

```diff
@@ -0,0 +1,63 @@
+import type { Core } from '@strapi/types';
+import { readFile } from 'node:fs/promises';
+import type { File } from 'formidable';
```

Comment:
This throws an error causing the build to fail, but I think you can use FileInfo from types which extends this 🤷